docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/index.md

```diff
@@ -225,11 +260,11 @@ In addition the ecommerce package has some contexts that can be enabled dependin
+| `snowplow__ecommerce_event_names`             | `['snowplow_ecommerce_action']`                                   | The list of event names that the Snowplow ecommerce package will filter on when extracting events from your atomic events table. If you have included any custom ecommerce events, feel free to add their event name in this list to include them in your data models.                                                                                                                                                                                                                                        |
 | `snowplow__number_checkout_steps`             | `4`                                   |  The index of the checkout step which represents a completed transaction. This is required to enable working checkout funnel analysis, and has a default value of 4.                                                                                                                                                                                                                                       |
-| `snowplow__number_subcategories`             | `4`                                   | The number of subcategories that exist on your website for products. This subcategories will be defined in the category field of the product context, and should be separated using the separator which is defined below. The default value for the number of subcategories is 4, and so 4 columns will be generated in the product tables created by this Snowplow dbt package.                                                                                                                                                                                                                                        |
+| `snowplow__number_category_level`             | `4`                                   | The number of subcategories that exist on your website for products. This subcategories will be defined in the category field of the product context, and should be separated using the separator which is defined below. The default value for the number of subcategories is 4, and so 4 columns will be generated in the product tables created by this Snowplow dbt package.                                                                                                                                                                                                                                        |
```

Comment:
I know I suggested literally this exact thing, but it should probably be a plural

Reply:
😂